src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(reject, 3000);
```

Comment:
should be message `First promise was rejected`

Reply:
still not fixed. there is no notification about rejecting the first promise
![image](https://user-images.githubusercontent.com/98764641/226310916-92e6203b-a250-4802-91a9-cbd7c25dbe36.png)